uptane-whitepaper.md

```diff
@@ -1 +1,109 @@
+
+Full verification provides better protection to those ECUs that have the memory and storage capacity to conduct the procedure. But, because even the weakest ECUs can be afforded some protection by the less resource-intensive partial method, the security of the system as a whole is improved.
+
+## **How Uptane continues to evolve to meet a changing marketplace**
+Uptane is very much a living technology. Over the past three years, as it has standardized its technology, a team of academic, government, and automotive industry personnel have identified key principles to guide the continuing development of its specification and its governing standards. Here are the key values these efforts have embraced and how these changes have manifested themselves.
+
+**Agility:** Agility in this sense refers to staying ahead of the curve on emerging trends in the industry and being able to respond accordingly. Uptane benefits here from the cross-section of experts that continue to contribute to both the [Standard](https://uptane.github.io/papers/uptane-standard.1.1.0.html) governing the technology and the [Deployment Best Practices](https://uptane.github.io/papers/uptane-deployment-best-practices-1.1.0.html) which compile real-world variations from OEMs and first-tier suppliers. Also, as an open source project, anyone can review the standard or the best practices and thus the technology benefits from ongoing feedback derived from real-world implementations.
+
+**Ease of adoption:** Early on in its development, a decision was made that Uptane’s specification would only stipulate implementation formats for safety critical components. This was done in part to allow OEMs and suppliers to adopt Uptane with minimal or no needed changes to legacy systems. In doing so, the Standards team evolved a new form for specifying Uptane that was named a dual-layer specification. This format separates interoperability functions, such as backwards compatibility, localization, and deployment, from those essential to reliability, security, and functionality. The latter group of features, which constitute the actual standard, make up the baseline layer for instructions, while all the elements required for interoperability are specified in a second layer, known as a Protocols, Operations, Usage, and Formats (POUF) document. By giving implementers the option of creating a [POUF](https://ssl.engineering.nyu.edu/papers/moore_pouf_2020.pdf),  Uptane can be adapted to the needs of existing implementations without requiring extensive modifications. It also makes it easier to add suppliers as needed without having to share proprietary designs.
```

Comment:
The implementation formats part here is weird in the first sentence.  I think this needs to be rephrased.  Perhaps explain the problem and the goal?

Reply:
Yes. I can fix this.
```suggestion
**Ease of adoption:** As mentioned earlier, Uptane never fights against deployment best practices and one component of that commitment  is that the adopter should not need to reconfigure its software update system just to integrate the framework. For this reason, the Uptane community made a decision early on not to specify data binding formats  or other protocols, operations, usage, and formats. Instead, the Standards team evolved a new form for specifying Uptane that separates interoperability functions, such as backwards compatibility, localization, and deployment, from those essential to reliability, security, and functionality. The latter group of features, which constitute the actual standard, make up the baseline layer for instructions, while all the elements required for interoperability are specified in a second layer, known as a Protocols, Operations, Usage, and Formats (POUF) document. By giving implementers the option of creating a [POUF](https://ssl.engineering.nyu.edu/papers/moore_pouf_2020.pdf),  Uptane can be adapted to the needs of existing implementations without requiring extensive modifications. It also makes it easier to add suppliers as needed without having to share proprietary designs.
```